docs/contribute/release_checklist.md

```diff
@@ -4,7 +4,7 @@ title: Release checklist
 permalink: /contributing/release-checklist
 ---
 
-This is instructions for preparing a release branch for Android Components Beta release and starting the next development cycle.
+These are instructions for preparing a Beta release branch for Android Components and starting the next Nightly development cycle.
```

Comment:
These instructions are not just for beta because you can use them for release too which are similar with the Gecko.kt parts. 🙂